FAQ_ptBR.md

```diff
@@ -0,0 +1,31 @@
+# Memo - Perguntas Frequentes
+
+- [Memo - Perguntas Frequentes](#memo---perguntas-frequentes)
+  - [O que é uma Coleção?](#o-que-é-uma-coleção)
+  - [O que é um Memo?](#o-que-é-um-memo)
+  - [O que é o nível de fixação?](#o-que-é-o-nível-de-fixação)
+
+## O que é uma Coleção?
+
+Uma coleção se trata de um grupo de [Memos](#o-que-é-um-memo).
+
+Fazendo uma analogia com [flashcards](https://en.wikipedia.org/wiki/Flashcard), ela é o que consideramos um Deck.
+
+## O que é um Memo?
+
+Um memo é um "desafio" individual de uma [Coleção](#o-que-é-uma-coleção).
+
+Fazendo uma analogia com [flashcards](https://en.wikipedia.org/wiki/Flashcard), é o que consideramos como uma Carta,
+esta na qual pertence à um Deck.
+
+## O que é o nível de fixação?
+
+O nível de fixação (traduzido de *memory recall*) pode ser definido como **a soma das estimativas que damos à
+probabilidade de um usuário ter memorizado todos os Memos de uma Coleção**. Este nível é calculado com base na
+[curva de esquecimento do SuperMemo](https://supermemo.guru/wiki/Forgetting_curve).
```

Comment:
What about instead
> **a soma das estimativas que damos à probabilidade de um usuário ter memorizado todos os Memos de uma Coleção**

We say like:
> **a soma das forças de memória do usuário para cada Memo, isto é, o quão forte é a memória do usuário sobre os Memos de uma Coleção**

SuperMemo uses the term [*Memory Strength*](https://supermemo.guru/wiki/Two_components_of_memory), which I personally think that makes a lot of sense

Reply:
But that's not what we estimate, we only use the recall given a forgetting curve. What we calculate is the `R - probability of recall (retrievability of memory)`, and not the strength of a memory. They may be similar, but I prefer to maintain the nomenclature defined by SuperMemo, which is our core reference for this implemenation.

Reply:
I will update given your suggestion nonetheless, given that the overall syntactic structure is better